asset-registry/src/lib.rs

```diff
@@ -0,0 +1,243 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+#![allow(clippy::large_enum_variant)]
```

Comment:
please fix the warning instead of suppress it. in this case you should box VersionedMultiLocation

Reply:
Done, but I think this warning was maybe a false positive, since the other items contain `AssetMetadata`, which has a `Option<VersionedMultiLocation>` field, so as far as I can tell the sizes should have been similar to each other

Reply:
It could be false negative. It is known issue that big enum of Call / Event can cause trouble.